Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -1777,9 +1777,33 @@ void CPacketHandler::Packet_Vehicle_InOut(NetBitStreamInterface& bitStream)
                             CClientPed* pJacked = pVehicle->GetOccupant(ucSeat);
 
                             // If it's the local player or syncing ped getting jacked, reset some stuff
-                            if (pJacked && (pJacked->IsLocalPlayer() || pJacked->IsSyncing()))
-                            {
-                                pJacked->ResetVehicleInOut();
+                            if (pJacked)
```

Comment:
It seems that there should be an opening bracket.

Reply:
In any case, nested conditional without brackets it is a kind of attribute of the bad code style.

Reply:
Fixed